### PR TITLE
regroup and clean est_abz_params

### PR DESCRIPTION
**Pull request by Eric-Sommer**
### What problem do you want to solve?

regroup and clean est_abz_params

- [x] some tests need corrections.

### CODE REVIEW

File: gettsim/taxes/abgelt_st.py [hunk 1]
```diff
@@ -11,8 +11,8 @@ def abgelt_st(tax_unit, eink_st_params, eink_st_abzuege_params):
             "abgst"
         ] * np.maximum(
             tax_unit["brutto_eink_5"]
-            - eink_st_abzuege_params["spsparf"]
-            - eink_st_abzuege_params["spwerbz"],
+            - eink_st_abzuege_params["sparerpauschbetrag"]
+            - eink_st_abzuege_params["sparer_werbungskosten_pausch"],
```
Comment (MaxBlesch) [inline gettsim/taxes/abgelt_st.py:15]: We should use either `pausch` or `pauschale`, not both. Also `werbungskostenpauschale` is one word

File: gettsim/taxes/abgelt_st.py [hunk 2]
```diff
@@ -24,8 +24,8 @@ def abgelt_st(tax_unit, eink_st_params, eink_st_abzuege_params):
                 tax_unit["brutto_eink_5_tu"]
                 - 2
                 * (
-                    eink_st_abzuege_params["spsparf"]
-                    + eink_st_abzuege_params["spwerbz"]
+                    eink_st_abzuege_params["sparerpauschbetrag"]
+                    + eink_st_abzuege_params["sparer_werbungskosten_pausch"]
```
Comment (MaxBlesch) [inline gettsim/taxes/abgelt_st.py:28]: Same as before.

File: gettsim/taxes/zve.py [hunk 3]
```diff
@@ -370,13 +376,14 @@ def _vorsorge_since_2010(tax_unit, params, soz_vers_beitr_params):
     # 'Basisvorsorge': Health and old-age care contributions are deducted anyway.
     sonstige_vors = 12 * (
         tax_unit["pflegev_beit_m"]
-        + (1 - params["vorsorg_krank_minder"]) * tax_unit["ges_krankv_beit_m"]
+        + (1 - params["vorsorg_kranken_minderung"]) * tax_unit["ges_krankv_beit_m"]
```
Comment (MaxBlesch) [inline gettsim/taxes/zve.py:379]: Either `vorsorg` or `vorsorge`.

File: gettsim/taxes/zve.py [hunk 4]
```diff
@@ -430,10 +439,12 @@ def vorsorge_pre_2005(tax_unit, params, soz_vers_beitr_params):
         )
         # Deduct a 'Grundhöchstbetrag' (1334€ in 2004),
         # or the actual expenses if lower (which is unlikely)
-        item_2 = np.minimum(params["grundbet"], vorsorg_rest)
+        item_2 = np.minimum(params["vors2004_grundhöchstbetrag"], vorsorg_rest)
```
Comment (MaxBlesch) [inline gettsim/taxes/zve.py:442]: Same. Consistency in `vorsorge`.

File: gettsim/taxes/abgelt_st.py (hunk 1 above)
Comment (Eric-Sommer) [reply gettsim/taxes/abgelt_st.py:15]: Changed. But a `_` every now and then can't hurt for such lung words.

File: gettsim/data/abgelt_st.yaml [hunk 5]
```diff
@@ -1,14 +1,11 @@
-abgst:
+abgeltungssteuer_satz:
```
Comment (MaxBlesch) [inline gettsim/data/abgelt_st.yaml:1]: The file is named abgelt_st. We should use that abbreviation also for the parameters.

**Pull request merged**